cluster/addons/fluentd-gcp/fluentd-gcp-configmap.yaml

```diff
@@ -375,7 +377,7 @@ data:
       buffer_queue_full_action block
       # Set the chunk limit conservatively to avoid exceeding the GCL limit
       # of 10MiB per write request.
```

Comment:
Want to sync this [comment](https://github.com/GoogleCloudPlatform/google-fluentd/blob/master/templates/etc/td-agent/td-agent.conf#L18) too?

Reply:
Good catch, thanks, done